### PR TITLE
 Optionalize 'related document or template' for proposals

### LINKED ISSUE #4435: Antragsdokument aus Dossier-Dokumente auswählen
Zurzeit ist es bei der Erstellung eines Antrages nur möglich eine neues Antragsdokument zu erstellen (generiert ab der ausgewählten Antragsvorlage). Neu soll es auch möglich sein, ein bestehendes Dokument aus dem aktuellen Dossier, als Antragsdokument auszuwählen.

- [ ] Zusätzliches Feld `Antragsdokument` im Antrag hinzufügen Formular
- [ ] Felder für Vorlage per Toggle anwählbar (entweder Antragsdokument oder Vorlagenauswahl)
bildschirmfoto_2018-06-11_um_15_29_56
- [ ] Validierung das entweder ein `Antragsdokument` oder eine `Antragsvorlage` ausgewählt wurden
- [ ] Verwenden des Antragsdokuments (Kopieren in Antrag)
- [ ] Zusätzliche Aktion `Antrag hinzufügen` bei Dokumentauflistungen
bildschirmfoto_2018-06-11_um_15_33_57

### PR DESCRIPTION
**Pull request by Rotonen**
* Not very happy with the null object
* `TableRadioWidget` can now sensibly be used as a non-required field
  * The user can cancel the selection
  * Upon an another validation mistake, a none selection is not persisted
    * Anyone know how to do that proper

Closes #4435

### CODE REVIEW

**Comment by Rotonen** [comment]
Ready for review, missing the error translation.

File: opengever/meeting/tests/test_proposal.py [hunk 1]
```diff
@@ -57,6 +57,19 @@ class TestProposal(IntegrationTestCase):
 
     features = ('meeting',)
 
+    @browsing
+    def test_create_proposal_visible_in_dossier_actions_for_manager(self, browser):
+        # XXX - no other user in the fixtures seems to be Manager, Contributor or Administrator on a dossier
```
Comment (deiferni) [inline opengever/meeting/tests/test_proposal.py]: It must be visible for `dossier_responsible`, otherwise you have a permission issue i'd say 🤔.

Comment (Rotonen) [reply opengever/meeting/tests/test_proposal.py]: It is likely I do have a permission issue. Per the rolemap the fixtures should be amended to grant `Contributor` for `dossier_responsible`?

Comment (Rotonen) [reply opengever/meeting/tests/test_proposal.py]: Mystery solved: I had a capitalisation mistake in the permission name.

**Comment by Rotonen** [comment]
Missing a translation for the mimetype validation of the proposal document, but otherwise ready for rereview.

**Pull request merged**